alertmanager/alerting.go

```diff
@@ -16,7 +16,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package promconfig
+package alertmanager
```

Comment:
That's a part of Prometheus configuration, not Alertmanager configuration. It should that in that package.

Alertmanager configuration files are there: https://github.com/prometheus/alertmanager/tree/master/config

Reply:
i thought it meant this https://github.com/prometheus/alertmanager/tree/master/config plus the existing alertmanager types, will revert this commit